oscar.py

```diff
@@ -1537,132 +1548,116 @@ class Time_commit_info(Clickhouse_DB):
         >>> t = Time_commit_info()
         >>> commits = t.commits_iter(1568656268)
         >>> c = commits.next()
         >>> type(c)
         <class 'oscar.Commit'>
         >>> c.parent_shas
         ('9c4cc4f6f8040ed98388c7dedeb683469f7210f5',)
-        '''
-        row_iter = self.query_select_iter('lower(hex(sha1))', self.tb_name, start, end)
-        for row in row_iter:
-            yield Commit(row[0])
+        """
+        for sha in self.commits_shas(start, end):
+            yield Commit(sha)
 
     def commits_shas(self, start, end=None):
-        ''' return a list of shas within the given time and date
-        >>> t = Time_commit_info()
-        >>> shas = t.commits_shas(1568656268)
-        >>> type(shas)
-        <type 'list'>
-        '''
-        rows = self.query_select('lower(hex(sha1))', self.tb_name, start, end)
-        return [row[0] for row in rows]
-
-    def commits_shas_iter(self, start, end=None):
-        ''' return a generator of all sha1 within the given time and date
+        """ return a generator of all sha1 within the given time and date
         >>> t = Time_commit_info()
-        >>> for sha1 in t.commits_shas_iter(1568656268):
+        >>> for sha1 in t.commits_shas(1568656268):
         ...     print(sha1)
-        ''' 
-        row_iter = self.query_select_iter('lower(hex(sha1))', self.tb_name, start, end)
-        for row in row_iter:
+        """
+        for row in self.query_select_iter(
+                'lower(hex(sha1))', self.tb_name, start, end):
             yield row[0]
-        
+
+
 class Time_project_info(Clickhouse_DB):
-    ''' Time_project_info class is initialized with table name and database host name
-        The default table name for projects is projects_all, and the default database name is localhost
-        This class contains methods to query for project data
-        The 'b2cPtaPkgR_all' table descrption is the following:
+    """ Time_project_info class is initialized with table name and database host
+    name. The default table name for projects is projects_all, and the default
+    database name is localhost. This class contains methods to query for project
+    data.
+        The 'projects_all' table descrption is the following:
         |___name___|______type_______|
-        | blob     | FixedString(20) |
-        | commit   | FixedString(20) |
-        | project  | String          |
+        | sha1     | FixedString(20) |
         | time     | UInt32          |
-        | author   | String          |
+        | blob     | FixedString(20) |
         | language | String          |
+        | repo     | String          |
+        | author   | String          |
         | deps     | String          |
-    '''
-    columns = ['blob', 'commit', 'project', 'time', 'author', 'language', 'deps']
+    """
+    columns = ['sha1', 'time', 'blob', 'language', 'repo', 'author', 'deps']
 
-    def __init__(self, tb_name='b2cPtaPkgR_all', db_host='localhost'):
```

Comment:
This should be the correct default table name.

Reply:
yes, there was a merge conflict I had to resolve manually and I overlooked these changes. I restored the last schema, can you please have a look again?